src/reducers/change-fixture.js

```diff
@@ -0,0 +1,10 @@
+var CHANGE_FIXTURE = require('../actions/change-fixture.js').type;
+
+module.exports = function(state = {}, action) {
+  switch (action.type) {
+    case CHANGE_FIXTURE:
+      return action.data || {};
```

Comment:
> the redux store should be a single object.

Yes, but an action shouldn't dictate the shape of the whole store, only part of it.

```js
Object.assign({}, state, { fixture: action.data });
```

The initial state should probably be `{ fixture: {} }`.